src/navigators/EditTextNavigator.tsx

```diff
@@ -1,13 +1,13 @@
-import { createStackNavigator } from 'react-navigation';
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react';
 
+import { Route } from 'app/consts';
 import { EditTextScreen } from 'app/screens/EditTextScreen';
 
-export const EditTextNavigator = createStackNavigator(
-  {
-    EditText: EditTextScreen,
-  },
-  {
-    headerMode: 'screen',
-    mode: 'modal',
-  },
+const Stack = createStackNavigator();
+
+export const EditTextNavigator = () => (
```

Comment:
I would consider making the structure of views / navigators much more flat (remove unnecessary navigators) and customize their animations per screen
```
const modalTransition = {
  cardStyleInterpolator: CardStyleInterpolators.forVerticalIOS,
  gestureDirection: 'vertical' as any,
};

...
<Stack.Screen
          name={Route.CreateAdSuccess}
          component={CreateAdSuccessScreen}
          options={{ ...modalTransition }}
        />
```